coding-concepts.js

```diff
@@ -37,23 +38,24 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: index elements start at zero - first element counted as zero, then one, two, ++. In this array, "Ruby" is the second element (first index)
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: SATURDAY, SUNDAY
+// b) Verify and explain: returned not a function - error - need to loop through array to add uppercase to each string in array
```

Comment:
Exactly, .toUpperCase() is a built-in methods for strings. Therefore to use the method on the values inside the array, you will have to iterate through the array. The method will work on the values then because the values are strings.